docs/reference/jinja2.md

```diff
@@ -92,7 +92,13 @@ Browsers will select the most appropriate image to load based on [responsive ima
 The [`sizes`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#sizes) attribute is essential unless you store the output of `srcset_image` for later use.
 
 ```html+jinja
-{{ srcset_image(page.header_image, "fill-{512x100,1024x200}", sizes="100vw", class="header-image") }}
+{{ srcset_image(page.photo, "width-{400,800}", sizes="(max-width: 600px) 400px, 80vw") }}
+```
+
+This outputs:
+
+```html
+<img srcset="/media/images/pied-wagtail.width-400_ENyKffb.jpg 400w, /media/images/pied-wagtail.width-800_ENyKffb.jpg 800w" src="/media/images/pied-wagtail.width-400_ENyKffb.jpg" alt="A pied Wagtail" sizes="(max-width: 600px) 400px, 80vw" width="400" height="300">
```

Comment:
@thibaudcolas correct me if I am wrong - `_ENyKffb` is a prefix added for duplicate images. If so, it would be cleaner/less confusing to remove it from the docs